src/services/accounts/AccountsStakingPayoutsService.spec.ts

```diff
@@ -0,0 +1,282 @@
+import { ApiPromise } from '@polkadot/api';
+import { ApiDecoration } from '@polkadot/api/types';
+import { DeriveEraExposure } from '@polkadot/api-derive/types';
+import { Option, u32 } from '@polkadot/types';
+import {
+	AccountId32,
+	BalanceOf,
+	EraIndex,
+	Hash,
+} from '@polkadot/types/interfaces';
+import {
+	PalletStakingEraRewardPoints,
+	PalletStakingStakingLedger,
+	PalletStakingValidatorPrefs,
+} from '@polkadot/types/lookup';
+import { Codec } from '@polkadot/types/types';
+import { BadRequest } from 'http-errors';
+
+import { sanitizeNumbers } from '../../sanitize/sanitizeNumbers';
+import { polkadotRegistryV9122 } from '../../test-helpers/registries';
+import { defaultMockApi } from '../test-helpers/mock';
+import {
+	mockDeriveNominatedExposure,
+	mockDeriveValidatorExposure,
+	mockEraRewardPoints,
+	mockLedger,
+} from '../test-helpers/mock/accounts';
+import stakingPayoutsResponse from '../test-helpers/responses/accounts/stakingPayout.json';
+import { AccountsStakingPayoutsService } from './AccountsStakingPayoutsService';
+
+/**
+ * Addresses and data below were taken from era 533 around block ~7,760,000,
+ * on runtime v9122 Polkadot.
+ *
+ * The real world data has been modified to scale to the unit tests to act as mock data.
+ * This test suite also uses polkadotRegistryV9122
+ */
+
+/**
+ * Acts as a placement variable for some tests where the era isn't an instrumental
```

Comment:
What do you mean by "placement variable"?

Reply:
So it's a placement variable for some of the mock test calls. We call a few of them in the test suite, and even though it doesn't need the `era` param to check the integrity of the test, we still need it to call the mock test call.

Reply:
So you mean it's a placeholder of sorts?

Reply:
Yes exactly, placeholder is the right term.